lib/resty/radixtree.lua

```diff
@@ -162,6 +163,41 @@ local function insert_route(self, opts)
     return true
 end
 
+local function insert_route_with_host(self, opts)
+    local path = opts.path
+    opts = clone_tab(opts)
+
+    if not self.disable_path_cache_opt
```

Comment:
`if not self.disable_path_cache_opt and opts.path_op == '=' then`

The single line code here is better.

Reply:
I just copy the style from insert_route